baggers.js

```diff
@@ -14712,6 +14711,69 @@ var data = {
         mail: 'julien.pauli@gmail.com',
       },
     },
+    {
+      since: '2018-06-09',
+      name: 'Xavier Blanc',
+      bio:
+        "Développeur (et accessoirement Professeur à l'Université de Bordeaux), toujours curieux des nouvelles approches de programmation (Clean Code, Craftmanship, DDD, BDD, Dette Technique, etc.).",
+      picture: 'https://avatars3.githubusercontent.com/u/10332041?s=460&v=4',
+      cities: ['Bordeaux', 'Paris'],
+      contacts: {
+        mail: 'xavier.blanc@u-bordeaux.fr',
+        websites: [
+          {
+            name: 'Web',
+            url: 'http://www.labri.fr/perso/xblanc/',
+          },
+          {
+            name: 'LinkedIn',
+            url: 'https://www.linkedin.com/in/xavier-blanc-3b9785a/',
+          },
+        ],
+      },
+      websites: [
+        {
+          name: 'Web',
+          url: 'http://www.labri.fr/perso/xblanc/',
+        },
+        {
+          name: 'LinkedIn',
+          url: 'https://www.linkedin.com/in/xavier-blanc-3b9785a/',
+        },
+        {
+          name: 'GitHub',
+          url: 'https://github.com/xblanc33',
+        },
+        {
+          name: 'YouTube',
```

Comment:
Although YouTube is not part of the official list, I'm +1 to add it. (http://data.brownbaglunch.fr/#websites)